src/app/app.module.ts

```diff
@@ -3,16 +3,31 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { HeaderComponent } from './components/header/header.component';
+import { FooterComponent } from './components/footer/footer.component';
+import { LogoComponent } from './components/logo/logo.component';
+import { CoursesListComponent } from './components/courses-list/courses-list.component';
+import { CourseComponent } from './components/courses-list/course/course.component';
+import { ManageCourseComponent } from './components/courses-list/manage-course/manage-course.component';
+import { LoginComponent } from './components/login/login.component';
+import { SearchBarComponent } from './components/search-bar/search-bar.component';
+import { BreadcrumbsComponent } from './components/breadcrumbs/breadcrumbs.component';
 
 @NgModule({
   declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    AppRoutingModule
+    AppComponent,
+    HeaderComponent,
+    FooterComponent,
+    LogoComponent,
+    CoursesListComponent,
+    CourseComponent,
+    ManageCourseComponent,
+    LoginComponent,
+    SearchBarComponent,
+    BreadcrumbsComponent,
   ],
+  imports: [BrowserModule, AppRoutingModule],
   providers: [],
-  bootstrap: [AppComponent]
+  bootstrap: [AppComponent],
```

Comment:
Please setup Prettier in your project to automatically resolve any formatting issues.